drivers/sensors/cirque_pinnacle.h

```diff
@@ -43,7 +37,10 @@ void            cirque_pinnacle_set_scale(uint16_t scale);
 #ifndef CIRQUE_PINNACLE_Y_RANGE
 #    define CIRQUE_PINNACLE_Y_RANGE (CIRQUE_PINNACLE_Y_UPPER - CIRQUE_PINNACLE_Y_LOWER)
 #endif
-
+#if !defined(POINTING_DEVICE_TASK_THROTTLE_MS)
+#    undef POINTING_DEVICE_TASK_THROTTLE_MS
```

Comment:
```suggestion
```

This branch can only be reached if `POINTING_DEVICE_TASK_THROTTLE_MS` wasn't defined before?

Reply:
removed - thanks!